extensions/amp-story/1.0/amp-story.js

```diff
@@ -97,6 +97,7 @@ import {debounce} from '../../../src/utils/rate-limit';
 import {dev, devAssert, user} from '../../../src/log';
 import {dict} from '../../../src/utils/object';
 import {findIndex} from '../../../src/utils/array';
+import {getAnalyticsService} from './story-analytics';
```

Comment:
nit: merge the two imports from `./story-analytics`

Reply:
Done